docker/Makefile

```diff
@@ -0,0 +1,27 @@
+SHELL=bash
+
+version=latest
+tag=$(version)
+cmd=
+user=directus
+registry=docker.io
+repository=directus/directus
+
+.PHONY: build
+
+build-images:
+	npm run build
+	npm run pack
+	docker build \
+		--build-arg VERSION=$(version) \
+		--build-arg REPOSITORY=$(repository) \
+		-t directus:temp \
+		-f ./Dockerfile \
+		..
+
+	docker tag directus:temp $(registry)/$(repository):$(version)
+	docker tag directus:temp $(registry)/$(repository):$(tag)
+	docker image rm directus:temp
+
+test-image:
+	docker run --rm -it $(registry)/$(repository):$(tag) $(cmd)
```

Comment:
Could you please rebase this on #6741? Thanks!

Reply:
OK should be correct now.

Reply:
Great, thanks! 😃